docker-compose.dev.yaml

```diff
@@ -1,32 +1,8 @@
 version: "3.8"
 
 services:
-  webserver:
-    restart: unless-stopped
-    image: did-auth-proxy-webserver
-    pull_policy: never
-    build:
-      context: nginx
-    ports:
-      - "127.0.0.1:8080:80"
-    networks:
-      dev-backend:
-    volumes:
-      - ./nginx/nginx.dev.conf:/opt/bitnami/nginx/conf/server_blocks/nginx.conf
-    extra_hosts:
-      - host.docker.internal:host-gateway
-
-  backend:
-    restart: unless-stopped
-    image: did-auth-proxy-backend
-    pull_policy: never
-    build:
-      context: backend-server
-    networks:
-      dev-backend:
-
   redis:
-    image: bitnami/redis:latest
+    image: bitnamilegacy/redis:latest
```

Comment:
`latest` tag is working but I read in the doc/reddit; `latest` tag will only be available on `bitnamisecure` registry. 

we should consider to update the tag with specific versions available on `bitnamilegacy` registry